backend/src/seed/factories/users.js

```diff
@@ -14,6 +14,8 @@ export default function create() {
         role: 'user',
         avatar: faker.internet.avatar(),
         about: faker.lorem.paragraph(),
+        // termsAndConditionsAgreedAt: new Date().toISOString(),
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T11:48:52Z" title="Friday, August 30th 2019, 1:48:52 pm +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -14,6 +14,8 @@ export default function create() {
         role: 'user',
         avatar: faker.internet.avatar(),
         about: faker.lorem.paragraph(),
+        termsAndConditionsAgreedAt: new Date().toISOString(),
```


Here you write it in the database for the seed or tests, but not if an user exactly agrees in the browser.